scalalib/src/mill/scalalib/publish/SonatypePublisher.scala

```diff
@@ -107,9 +111,7 @@ class SonatypePublisher(uri: String,
     }
   }
 
-  private def awaitRepoStatus(status: String,
-                              stagingRepoId: String,
-                              attempts: Int = 20): Unit = {
+  private def awaitRepoStatus(status: String, stagingRepoId: String, attempts: Int = 20): Unit = {
```

Comment:
Let's set scalafmt's max line length to 80 or 90 instead of 100

Reply:
@lihaoyi default is 80 with a comment:

```
Keep in mind that 80 characters fit perfectly on a split laptop screen with regular resolution.
Github mobile view only shows 80 characters and sometimes you might review code on your phone.
```

So should be use 80 then?